### PR TITLE
Added scripts for Keep Dashboard installation

### PR DESCRIPTION
**Pull request by nkuba**
The scripts can be used for Keep Dashboard installation and run.
To install the dashboard execute: `./install-keep-dashboard.sh`
To run the dasbhoard execute `./run-keep-dashboard.sh`

The install script is also a part of main `install.sh` and is executed after tBTC installation.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -132,6 +132,13 @@ end-to-end tests. Before you start interacting, make sure you:
 - Have a working relay maintainer (`run-testnet-relay.sh`)
 - Have 1 `keep-core` and 3 `keep-ecdsa` clients up and running
 
+== Keep Dashboard dApp
+
+To run the Keep Dashboard dApp invoke:
+```
+./run-keep-dashboard.sh
```
Comment (r-czajkowski) [inline README.adoc:139]: I think we miss this script, or I just can't see this file 🤔 .

Comment (nkuba) [reply README.adoc:139]: Thanks for finding this out! I just pushed the file.

File: install-keep-dashboard.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+printf "${LOG_START}Preparing keep-core artifacts...${LOG_END}"
+
+cd $WORKDIR/keep-core/solidity
+ln -sf build/contracts artifacts
+
+printf "${LOG_START}Preparing keep-ecdsa artifacts...${LOG_END}"
+
+cd $WORKDIR/keep-ecdsa/solidity
+ln -sf build/contracts artifacts
```
Comment (r-czajkowski) [inline install-keep-dashboard.sh:22]: In the keep token dashboard, we use tbtc contracts as well. I think we need symlink `artifacts` to `build/contracts`.

```
cd $WORKDIR/tbtc/solidity
ln -sf build/contracts artifacts
```

Comment (nkuba) [reply install-keep-dashboard.sh:22]: Thanks, I just added it.

**Comment by nkuba** [comment]
@r-czajkowski can we merge it?

**Pull request merged**